action.yml

```diff
@@ -53,6 +53,14 @@ inputs:
   image_family:
     description: The image family for the operating system that the boot disk will be initialized with.
     required: false
+  preemptible:
+    description: Use GCE preemptible VM instance; https://cloud.google.com/compute/docs/instances/preemptible
+    default: false
+    required: true
+  ephemeral:
+    description: Set GitHub runner to be ephemeral; https://docs.github.com/en/actions/hosting-your-own-runners/autoscaling-with-self-hosted-runners#using-ephemeral-runners-for-autoscaling
+    default: false
```

Comment:
And this one could actually be true by default right? Since the gce-runners are always ephemeral one job runners. I'm +1 to that, and sorry for not flagging it in the 1st review. wdyt?

Reply:
Are they actually just one job runners though? I thought this Action allowed multiple jobs to be run, no?

Reply:
@Manouchehri the way we have been using it, and the way the examples are setup, it's used as a one job runner because we pass `runs-on` https://github.com/related-sciences/gce-github-runner/blob/f8026318c0af4b968bd50f1bb23729c548112a39/.github/workflows/test.yml#L26 to **unique** label, which essentially forces one job only, in fact it must be the job from the workflow the runner was created it.